homeworks/07-mars-viewer/src/components/marsViewer/gallery/galleryAllPhotos.tsx

```diff
@@ -0,0 +1,65 @@
+import {Dispatch} from 'react'
+import {useDispatch, useSelector} from 'react-redux'
+
+import {MarsViewerState} from "../../../types/state"
+import {changeFavouriteStatus} from './galleryUtils'
+
+import Style from './gallery.module.css'
+
+
+export const MarsViewerAllPhotos = (): JSX.Element => {
+    const dispatch: Dispatch<any> = useDispatch()
```

Comment:
better create one typed hook like `useAppDispatch` in this example and reuse it across the app https://redux-toolkit.js.org/tutorials/typescript#use-typed-hooks-in-components